jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminListPresenter.java

```diff
@@ -58,9 +63,73 @@ public FilterSettings createTableSettingsPrototype() {
     }
 
     public FilterSettings createAdminTabSettings() {
-        //Filter status Admin
-        return createStatusSettings(HUMAN_TASKS_WITH_ADMIN_DATASET,
-                                    new ArrayList<>(getStatusByType(TaskUtils.TaskType.ADMIN)));
+        FilterSettingsBuilderHelper builder = FilterSettingsBuilderHelper.init();
+        builder.initBuilder();
+
+        builder.dataset(HUMAN_TASKS_WITH_ADMIN_DATASET);
+        List<Comparable> status = new ArrayList<>(getStatusByType(TaskUtils.TaskType.ADMIN));
+        if (status != null) {
+            builder.filter(COLUMN_STATUS,
+                           equalsTo(COLUMN_STATUS,
+                                    status));
+        }
+        builder.group(COLUMN_TASK_ID);
+
+        addCommonColumnSettings(builder);
+        return builder.buildSettings();
+    }
+    
+    @Override
+    protected void addCommonColumnSettings(FilterSettingsBuilderHelper builder) {
+        builder.setColumn(COLUMN_ACTIVATION_TIME,
```

Comment:
I guess you can just add the error column in here and call super to add the remaining?